### ISSUE TITLE
Flow getters and setters in interface definition cause parsing errors

### ISSUE THREAD

**Issue by matthewtung** [gajus/eslint-plugin-flowtype#306]
Getters and setters are enabled by default since Flow 0.62 (see flow [issues/5594](https://github.com/facebook/flow/issues/5594)).

However defining these in an interface causes eslint to fail:
```js
interface FooBar {
  get Foo(): Object,
}
```

```js
src/test.js
  2:7  error  Parsing error: Unexpected token, expected :

  1 | interface FooBar {
> 2 |   get Foo(): Object,
    |       ^
  3 | }
```
Similar example from Flow repository: https://github.com/facebook/flow/blob/master/tests/unsafe_getters_setters/unsafe_getters_setters.js

Wrapping the interface with `/* eslint-disable */` does not help either, I haven't found a way to use get/set in Flow interfaces without eslint failing yet.

**Comment by matthewtung** [comment]
Wrong repository. :)
